### PR TITLE
tachyon 0.6.0

### PR DESCRIPTION
**Pull request by kashif**

### CODE REVIEW

File: Library/Formula/tachyon.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Tachyon < Formula
+  homepage "http://tachyon-project.org/"
+  url "https://github.com/amplab/tachyon/releases/download/v0.6.0/tachyon-0.6.0-bin.tar.gz"
+  sha1 "74c17c9fb186740246cc23953c8811c47bdf699e"
```
Comment (tdsmith) [inline Library/Formula/tachyon.rb]: Our new policy is to use sha256 for new objects; can you update the checksum, please?

File: Library/Formula/tachyon.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  def install
+    mkdir_p "#{etc}/tachyon"
```
Comment (tdsmith) [inline Library/Formula/tachyon.rb]: Suspect `(etc/"tachyon").install` should do this for you.

**Comment by tdsmith** [comment]
lgtm aside from nits above

**Comment by kashif** [comment]
@tdsmith ok done

**Comment by tdsmith** [comment]
Awesome; thanks a lot and sorry for the delay.

**Pull request closed without merging**